src/main/java/org/folio/inventory/validation/ItemMarkAsWithdrawnValidators.java

```diff
@@ -1,34 +1,27 @@
 package org.folio.inventory.validation;
 
+import static java.util.EnumSet.of;
 import java.util.EnumSet;
 import java.util.concurrent.CompletableFuture;
 
 import org.folio.inventory.domain.items.Item;
 import org.folio.inventory.domain.items.ItemStatusName;
-import org.folio.inventory.exceptions.NotFoundException;
 import org.folio.inventory.exceptions.UnprocessableEntityException;
 import org.folio.inventory.support.CompletableFutures;
 import org.folio.inventory.support.http.server.ValidationError;
 
 public final class ItemMarkAsWithdrawnValidators {
```

Comment:
I don't think the `item` prefix adds much to this name

```suggestion
public final class MarkAsWithdrawnValidators {
```